scene/resources/resource_format_text.cpp

```diff
@@ -388,6 +388,55 @@ Ref<PackedScene> ResourceLoaderText::_parse_node_tag(VariantParser::ResourcePars
 
 			error = VariantParser::parse_tag(&stream, lines, error_text, next_tag, &parser);
 
+			if (error) {
+				if (error != ERR_FILE_EOF) {
+					_printerr();
+					return Ref<PackedScene>();
+				} else {
+					error = OK;
+					return packed_scene;
+				}
+			}
+		} else if (next_tag.name == "override") {
+			NodePath path = next_tag.fields["path"];
+			auto o_data = packed_scene->get_state()->gen_override_data();
+			HashSet<StringName> path_properties;
+			o_data.path = path;
+			while (true) {
+				String assign;
+				Variant value;
+
+				error = VariantParser::parse_tag_assign_eof(&stream, lines, error_text, next_tag, assign, value, &parser);
+
+				if (error) {
+					if (error == ERR_FILE_MISSING_DEPENDENCIES) {
+						// Resource loading error, just skip it.
+					} else if (error != ERR_FILE_EOF) {
+						ERR_PRINT(vformat("Parse Error: %s. [Resource file %s:%d]", error_names[error], res_path, lines));
+						return Ref<PackedScene>();
+					} else {
+						error = OK;
+						packed_scene->get_state()->add_override(o_data);
+						return packed_scene;
+					}
+				}
+
+				if (!assign.is_empty()) {
+					StringName assign_name = assign;
+					auto o_prop = packed_scene->get_state()->gen_property();
+					o_prop.name = assign_name;
+					o_prop.value = packed_scene->get_state()->add_value(value);
+					o_data.properties.push_back(o_prop);
+				} else if (!next_tag.name.is_empty()) {
+					break;
+				}
+			}
+
+			packed_scene->get_state()->add_override(o_data);
+			continue;
```

Comment:
This `continue` is invalid, it just skips everything after, is this meant to be a draft?